### DISCUSSION TITLE
I can't get anti-kick to work

### DISCUSSION THREAD

**Discussion by ghost** [winstxnhdw/lc-hax#173] [Q&A] [answered]
First of all, I love this cheat. Thank you for making it public and free.

I just can never get anti-kick to work, it does show it is enabled but from what I can see I only become nameless and people can still kick me.

**Comment by winstxnhdw** [comment, marked as answer]
Hey @cupcakekid14, anti-kick doesn't always work. There's some caveats to it. It works best if you are the last player joining or leaving, because the host won't be able to resolve your Steam ID. There's a new `/invis` command that will be paired with this in the future as a bandaid solution.

**Reply by ghost** [reply]
Oh okay, thank you. Looking forward to that.

**Reply by winstxnhdw** [reply]
I have just added it in latest. Now using anti-kick will also toggle invisibility.
